cartridge/lua-api/get-topology.lua

```diff
@@ -238,7 +238,7 @@ local function get_topology()
                 uri = m.uri,
                 uuid = '',
                 status = 'unconfigured',
-                message = m.payload.error or m.payload.warning or '',
+                message = m.payload.state or '',
```

Comment:
Don't understand how it's related to current patch. Please describe.

Reply:
It's necessary for rejoin_test. Both `payload.error` and `payload.warning` are obsolete fields from 1.0. In cartridge v2 the `payload.state` is used instead, but this place wasn't edited in time.

Reply:
Seems this change should be tested separately